### DISCUSSION TITLE
How to use Tailwind colors with nested elements inside image?

### DISCUSSION THREAD

**Discussion by Anindya088** [zauberzeug/nicegui#2064] [Q&A] [answered]
### Question

Hi All,

When I am nesting some row/columns inside an image, I realized I can use the normal colors (ex: bg-red) or Quasar colors (ex: bg-red-6) but not the Tailwind colors (ex: bg-red-600). 

Ex: this is the code I was trying:
```python
with ui.image('https://picsum.photos/id/377/640/360').classes('w-[500px]'):
    ui.column().classes('w-full h-full bg-red-600')

ui.column().classes('w-[500px] h-[50px] bg-red-600')
```
I am getting this output:
![image](https://github.com/zauberzeug/nicegui/assets/65127974/15ec0e32-2808-4f84-8710-3360073fa34c)

If instead of ```bg-red-600```, the Tailwind version, I use just normal ```bg-red``` or any other Quasar version it works. As in the image is covered with that color. But for Tailwind colors its not happening.

Ex. If I use the Quasar ```red-2``` then I get this:
![image](https://github.com/zauberzeug/nicegui/assets/65127974/7a941e3d-0dd8-4c91-8fe2-116547d3838b)

Can you advise on how to properly use Tailwind colors for elements inside an image please?

Thanks,
Anindya

**Comment by falkoschindler** [comment, marked as answer]
I'm can't say why exactly Tailwind classes are overruled by Quasar, but enforcing the color with an "important" attribute seems to help:
```py
with ui.image('https://picsum.photos/id/377/640/360').classes('w-[500px]'):
    ui.column().classes('w-full h-full !bg-red-600 opacity-50')
```
Note that I added "opacity-50" to give the column some transparency. This is probably done automatically by Quasar when using Quasar colors.

**Reply by Anindya088** [reply]
Hi @falkoschindler , this is perfect, so just need to make it "important" -- awesome.

Thanks so much for the quick help

Anindya
